### DISCUSSION TITLE
Support for RestFulApiModelingLanguage - .raml file

### DISCUSSION THREAD

**Discussion by tsutaz** [sbarex/SourceCodeSyntaxHighlight#189] [Q&A]
Hi, Is it possible to add support for .raml files? (RestfulApiModelingLanguage)
My os x: macOS Moneterey Version 12.5 - Apple M1 Pro 
This is the output from the MDLS command is attached.
[mdlsOutput.txt](https://github.com/sbarex/SourceCodeSyntaxHighlight/files/9469432/mdlsOutput.txt)

This is a link to RAML.org: [RAML Org](https://raml.org/)

Thanks
Tsutaz

**Comment by sbarex** [comment]
it doesn't seem to me that it is a language supported by the rendering engine [Highlight](http://www.andre-simon.de/doku/highlight/en/langs.php).

If this language is derived from another supported language at most it would be possible to add partial support.
